src/core/index.ts

```diff
@@ -132,6 +140,18 @@ export class Circomkit {
     return await readR1CSInfo(this.path.ofCircuit(circuit, 'r1cs'));
   }
 
+  /** Returns human-readable constraint formula array. */
+  async constraints(circuit: string) {
+    const r1csPath = this.path.ofCircuit(circuit, 'r1cs');
+    const symPath = this.path.ofCircuit(circuit, 'sym');
+    // @ts-ignore
+    const r1csFile = await import('r1csfile');
```

Comment:
If this dependency is loaded synchronously at the top of the file, when I use the witnessTester to parse the constraints, I get this error:

```

(node:1210393) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)

 Exception during run: /home/ben/circomkit/node_modules/web-worker/node.js:17
import URL from 'url';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at wrapSafe (node:internal/modules/cjs/loader:1281:20)
    at Module._compile (node:internal/modules/cjs/loader:1321:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1416:10)
    at Module.load (node:internal/modules/cjs/loader:1208:32)
    at Module._load (node:internal/modules/cjs/loader:1024:12)
    at cjsLoader (node:internal/modules/esm/translators:348:17)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:297:7)
    at ModuleJob.run (node:internal/modules/esm/module_job:222:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async formattedImport (/home/ben/ntru-circom/node_modules/mocha/lib/nodejs/esm-utils.js:9:14)
    at async exports.requireOrImport (/home/ben/ntru-circom/node_modules/mocha/lib/nodejs/esm-utils.js:42:28)
    at async exports.loadFilesAsync (/home/ben/ntru-circom/node_modules/mocha/lib/nodejs/esm-utils.js:100:20)
    at async singleRun (/home/ben/ntru-circom/node_modules/mocha/lib/cli/run-helpers.js:162:3)
    at async exports.handler (/home/ben/ntru-circom/node_modules/mocha/lib/cli/run.js:375:5)

```

With the dynamic import, I can both print constraints through the witness tester and the cli without error.